config.proto

```diff
@@ -48,7 +48,7 @@ message Opa {
     google.protobuf.StringValue address = 1;
 
     // poll period in seconds to query OPA service
-    google.protobuf.Int32Value pollPeriod  = 2;
+    google.protobuf.Int32Value pollPeriodInSecs  = 2;
```

Comment:
Could we leave out the `In` and use `pollPeriodSeconds`. I haven't seen `Secs` anywhere.

Reply:
I think it is a matter of preference: `HT_REPORTING_OPA_POLL_PERIOD_SECONDS` vs `HT_REPORTING_OPA_POLL_PERIOD_IN_SECS`. I think the `in` adds readability but not 100% `secs` is meaningful enough. Thoughts @davexroth @mohit-a21

Reply:
I've seen and used secs, mins, hrs in the past.  But maybe in this case let's use `pollPeriodSeconds` etc as it's a little simpler.